core/block_validator.go

```diff
@@ -168,6 +174,16 @@ func (v *BlockValidator) ValidateState(block *types.Block, statedb *state.StateD
 // to keep the baseline gas close to the provided target, and increase it towards
 // the target if the baseline gas is lower.
 func CalcGasLimit(parent *types.Header, gasCeil uint64) uint64 {
+	// No Gas for TimeToStartTx days worth of zone blocks, this gives enough time to
+	// onboard new miners into the slice
+	if parent.NumberU64(common.ZONE_CTX) < params.TimeToStartTx {
```

Comment:
Are inbound ETXs allowed to enter the zone during this time?

Reply:
no they are not, we have to prevent them in both opetx and createetx like you mentioned